package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "cordova-plugin-camera",
-  "version": "6.0.1-dev",
+  "name": "@zenput/cordova-plugin-camera",
+  "version": "6.0.1-dev-zp1",
```

Comment:
Be advised that this is not a new fork; #2 overwrote our `@zenput/` prefix, so I'm just adding it back with a new version number 🙂